Documentation/gettingstarted/k8s-install-openshift-okd.rst

```diff
@@ -69,7 +69,7 @@ And set ``networkType: Cilium``:
 
 .. code-block:: shell-session
 
-   sed -i "s/networkType: .*/networkType: Cilium/" "${CLUSTER_NAME}/install-config.yaml"
+   sed -i.b "s/networkType: .*/networkType: Cilium/" "${CLUSTER_NAME}/install-config.yaml"
```

Comment:
I've got mixed feelings about this one. It will leave backup files behind, and it's also non consistent with the other calls to `sed -i` which we have at other locations in the docs, or even in test/, contrib/, install/ (not saying that's a good thing, but if we change them I'd as well have them changed altogether).

Reply:
If we were to change it everywhere we might not go for `sed -i.b`, maybe something with `perl` instead if it's available on most platforms? Or `sed > tmp; mv tmp orig`

Reply:
Yeah not sure which one I would go with, both seem like ok choices, maybe the redirect sed into tmp file approach just because I'm more familiar with sed but it does feel a bit clunky.